ext.php

```diff
@@ -67,6 +70,26 @@ protected function check_php_version()
 		return $this;
 	}
 
+	/**
+	 * Check the installed PHP extensions meet this extension's requirements.
+	 *
+	 * @return \phpbb\webpushnotifications\ext
+	 */
+	protected function check_php_requirements()
+	{
+		$extensions = ['curl', 'mbstring', 'openssl'];
+
+		foreach ($extensions as $extension)
```

Comment:
I'd even simplify it and do just
```suggestion
		foreach (['curl', 'mbstring', 'openssl'] as $extension)
```
to get rid of `$extensions` array.